src/Logging/Logging.Testing/src/Xunit/LoggedTestRunner.cs

```diff
@@ -97,6 +103,40 @@ private async Task<decimal> InvokeTestMethodAsync(ExceptionAggregator aggregator
             return totalTime;
         }
 
+        private async Task<decimal> RunRepeatTestInvoker(ExceptionAggregator aggregator, ITestOutputHelper output, bool collectDump, RepeatAttribute repeatAttribute)
+        {
+            var repeatContext = new RetryContext()
```

Comment:
nit: Maybe use a different context? `RepeatContext` instead of `RetryContext`? Even though they would be similar let's not cause confusion.